mp/src/game/client/c_baseanimating.cpp

```diff
@@ -4454,6 +4486,22 @@ void C_BaseAnimating::RagdollMoved( void )
 	m_pRagdoll->GetRagdollBounds( mins, maxs );
 	SetCollisionBounds( mins, maxs );
 
+#ifdef NEO
+	if (GetOldOrigin() != vec3_origin)
+	{
+		if (m_flLastOriginChangeTime != gpGlobals->curtime)
+		{
+			SetOldVelocity((GetAbsOrigin() - GetOldOrigin()) / (gpGlobals->curtime - m_flLastOriginChangeTime));
```

Comment:
Earlier I had oldVelocity be a float, but since technically a velocity is speed with direction and all the other velocities already are vectors, I decided to also make oldVelocity a vector. DistTo would return a float